mirrord/cli/src/error.rs

```diff
@@ -472,6 +472,12 @@ pub(crate) enum CliError {
     ))]
     NestedExec,
 
+    #[error("Database branch error: {0}")]
+    #[diagnostic(help(
+        "Make sure the database branch exists and the operator is running correctly."
+    ))]
+    DbBranchNotFound(String),
```

Comment:
Mentioned in the commands methods, I think we don't need this error, i.e. destroy command should be idempotent.